src/onion_messenger.rs

```diff
@@ -199,18 +199,29 @@ impl LndkOnionMessenger {
         let peers_sender = sender.clone();
```

Comment:
Can you change the commit prefix to "messenger" so it's more aligned with previous commits? Historically, the "server" prefix has been used for changes to the grpc server, while "messenger" is for changes touching the onion messenger code.